app/controllers/ReturnObligationsController.scala

```diff
@@ -51,8 +55,12 @@ class ReturnObligationsController @Inject()(val messagesApi: MessagesApi,
       )))
   }
 
-  private[controllers] def handleReturnObligations(user: User)(implicit hc: HeaderCarrier): Future[VatReturnObligations] = {
-    returnsService.getReturnObligationsForYear(user, 2017).map {
+  private[controllers] def validateSearchYear(year: Int, upperBound: Int = LocalDate.now().getYear) = {
```

Comment:
I don't think there's much point in having `upperBound` as a parameter only to default it when the method is only called once